bot/constants.py

```diff
@@ -236,6 +236,14 @@ class Colours(metaclass=YAMLGetter):
     soft_orange: int
 
 
+class DuckPond(metaclass=YAMLGetter):
+    section = "duck_pond"
+
+    ducks_required: int
+    duck_custom_emojis: List[int]
+    duck_pond_channel: int
```

Comment:
Prefixing with `duck` for some of these is redundant since the whole attribute is already under the `DuckPond` namespace. How about something like this?

```py
    threshold: int
    custom_emojis: List[int]
    channel: int
```